common/model/identity.js

```diff
@@ -1,5 +1,5 @@
 export const IdentityType = {
-  UNKNOWN: 'unknown',
+  // UNKNOWN: 'unknown',
```

Comment:
I'm fine with removing it if we don't want to use that

Reply:
Ok, I'll keep it uncommented.

Reply:
I'm fine with completely removing it, I don't think it's used or we want to support `UNKNOWN` types